pxtlib/package.ts

```diff
@@ -167,6 +197,23 @@ namespace pxt {
                         return undefined;
                     pxt.debug('downloading ' + verNo)
                     return this.host().downloadPackageAsync(this)
+                        .then(() => {
+                            if (deps) {
+                                let files = (this as any)._editorPkg.files;
```

Comment:
this seems to be too late. if you modify the dependencies after they are being downloaded, how will makecode be able to resolve them?

Reply:
I've changed it to update the config before. I had tried this before, but the config was getting overwritten. I've now changed it so if there is already empty files added, it just doesn't overwrite them.